target/linux/qualcommax/ipq807x/base-files/etc/hotplug.d/firmware/10-ath10k-caldata

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+[ -e /lib/firmware/$FIRMWARE ] && exit 0
+
+. /lib/functions/caldata.sh
+
+board=$(board_name)
+
+case "$FIRMWARE" in
+"ath10k/pre-cal-pci-0001:01:00.0.bin")
+	case $board in
+	linksys,mx5300)
+		caldata_extract "0:art" 0x33000 0x2f20
+		ath10k_patch_mac $(macaddr_add $(get_mac_label) 3)
+		ln -sf /lib/firmware/ath10k/pre-cal-pci-0001\:01\:00.0.bin \
```

Comment:
Sorry, but WTF is this?

Precal data is not BDF, I have seen this craziness being done before but its not valid.
Also, for ath10k you can load the pre-cal and caldata via NVMEM

Reply:
I know it's set up the wrong way at the moment.
`pre-calibration` is a data saved on the `art` partition and `calibration` is the BDF file included in the OEM firmware? Or maybe the other way around?
But if we want to change the MAC address outside DTS, this is probably how it should be done?

Reply:
Sorry but no, this is completely wrong as you are pretending that pre-cal data is BDF, and it's a completely different thing.

Remove the symbolic link please.

Reply:
The file has been deleted.